### PR TITLE
Use latest version of Cypress for npm run test:ci script

### PR DESCRIPTION
**Pull request by sareh**
Resolves #NUMBER

**Overall change:**
Use latest version of Cypress for script `npm run test:ci` script - which is run on CI

**Code changes:**

- Use latest version of Cypress for npm run test:ci script

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by karinathomasbbc — APPROVED** [review]

**Review by amywalkerdev — APPROVED** [review]

**Comment by sareh** [comment]
Test failed on CI step `cypress:ci` with this: 
```
The Test Runner unexpectedly exited via a exit event with signal SIGILL

Please search Cypress documentation for possible solutions:

https://on.cypress.io

Check if there is a GitHub issue describing this crash:

https://github.com/cypress-io/cypress/issues

Consider opening a new issue.

----------

Platform: linux (Debian - 9.12)
Cypress Version: 4.4.0
[?25hnpm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! simorgh@1.0.0 cypress:ci: `rm -rf cypress/results; xvfb-run -a cypress run --reporter cypress-multi-reporters --reporter-options configFile=cypress/reporter-config.json`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the simorgh@1.0.0 cypress:ci script.

```

**Review by DenisHdz — APPROVED** [review]

**Comment by j-pendlebury** [comment]
Apparently `Cypress@4.4.1` fixes this issue so I can add this to my dependency update PR https://github.com/bbc/simorgh/pull/6251

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "start": "NODE_ENV=production node build/server.js",
     "storybook": "start-storybook -p 9001 -s .storybook/static -c .storybook",
     "test": "npm run test:lint && npm run test:dependencies && npm run test:unit && npm run build -- --hide-modules --colors && run-p --race start amp:validate && run-p --race start test:integration:ci",
-    "test:ci": "npm run start & npm install --no-save cypress@3.4.1 bbc-a11y@latest npm-run-all@latest puppeteer@latest audit-ci@latest && npm run test:integration:ci && run-p cypress:ci test:puppeteer && run-p bbcA11y:ci lighthouse && audit-ci --low",
+    "test:ci": "npm run start & npm install --no-save cypress@4.4.0 bbc-a11y@latest npm-run-all@latest puppeteer@latest audit-ci@latest && npm run test:integration:ci && run-p cypress:ci test:puppeteer && run-p bbcA11y:ci lighthouse && audit-ci --low",
```
Comment (sareh) [inline package.json]:
```suggestion
    "test:ci": "npm run start & npm install --no-save cypress@4.4.1 bbc-a11y@latest npm-run-all@latest puppeteer@latest audit-ci@latest && npm run test:integration:ci && run-p cypress:ci test:puppeteer && run-p bbcA11y:ci lighthouse && audit-ci --low",
```

**Comment by sareh** [comment]
@jblaut Thank you, Jamie. I've updated this PR. Hopefully it'll pass on CI this time.

**Comment by amywalkerdev** [comment]
Still failing with `The Test Runner unexpectedly exited via a exit event with signal SIGTRAP`

**Comment by j-pendlebury** [comment]
New version is out https://github.com/cypress-io/cypress/releases/tag/v4.5.0, however it still breaks with `SIGTRAP` error

**Comment by j-pendlebury** [comment]
New version of Cypress is out now (https://github.com/cypress-io/cypress/releases/tag/v4.6.0), so testing to see if it causes the same issues

**Comment by j-pendlebury** [comment]
Ran Jenkins 3 times on this PR and all 3 times I got the SIGTRAP error

**Comment by thekp** [comment]
Small Update:

Simon and I tried bumping Cypress to it's latest version in this PR: https://github.com/bbc/simorgh/pull/6836#issuecomment-642817703 but hit a wall. We could not get it to work in our Travis CI.

**Comment by amywalkerdev** [comment]
Created an issue to track this instead, #7075

**Pull request closed without merging**